src/terminal.c

```diff
@@ -3033,7 +3039,15 @@ color2index(VTermColor *color, int fg, int *boldp)
     {
 	// Use the color as-is if possible, give up otherwise.
 	if (color->index < t_colors)
-	    return color->index + 1;
+	{
+	    uint8_t index = color->index;
+#ifdef MSWIN
+	    if (index < 16) {
+		index = ansi_to_cterm_nr16[index];
+	    }
```

Comment:
There are coding style issue.

```suggestion
	    if (index < 16)
		index = ansi_to_cterm_nr16[index];
```

Reply:
Thank you @h-east 👍